customizations/autoscaling/config.yaml

```diff
@@ -0,0 +1,112 @@
+accountsWww:
+  ## accounts-www autoscaling configuration
```

Comment:
I think I would delete the generic comments here because they doesn't add too much value.

That apply to all the file.